### ISSUE TITLE
Bottom navigation is visually confusing.

### ISSUE THREAD

**Issue by techwraith** [mikeal/gatherbugs#44]
The selected state and the unselected state feel like they should be switched. I constantly click the wrong one first. Generally speaking, the active tab should be the one that looks visually different from all other buttons in the nav bar.

**Comment by apage43** [comment]
This confuses the ever loving crap out of me too. +1.

**Comment by max-mapper** [comment]
what do ya'll think of this?

![](http://i.imgur.com/oPwyx.png)

**Comment by techwraith** [comment]
Yep, that'd do it! (assuming that you're showing the following page)
